environment.yaml

```diff
@@ -0,0 +1,21 @@
+# A conda environment for CoCo coding sessions.
+#
+# Usage:
+#   $ conda env create --file=environment.yaml
+#   $ source activate coco
+
+name: coco
+channels:
+  - conda-forge
+dependencies:
+  - python =3
+  - pip
+  - numpy
+  - scipy
+  - pandas
+  - matplotlib
+  - jupyterlab
+  - xarray
+  - pygmt
+  - pyproj
+  - seaborn
```

Comment:
This time we do not use these packages. Wouldn't it be better if they weren't there?

Reply:
Ah, true--they're mentioned in the last notebook for further study, so I included them in the environment in case someone wanted to try them. The tradeoff is that it makes the environment more complicated than it needs to be. Given this, let me know what you think & I'll change it based on your decision.

Reply:
I prefer not to put them. If we use them in the second part of CoCo we can add them.
Also sometimes pyproj has installation problems

Reply:
Got it. I'll take them out. And that's a good point on pyproj; gdal has been difficult for years.

Reply:
Done!